packages/toolpad-app/src/toolpad/AppEditor/HierarchyExplorer/CreateCodeComponentNodeDialog.tsx

```diff
@@ -62,6 +62,9 @@ export default function CreateCodeComponentDialog({
     event.target.select();
   }, []);
 
+  const inputErrorMsg = name ? appDom.validateNodeName(name, 'a code component name') : null;
```

Comment:
i guess it might be a bit more semantic to validate on change or in an effect that only runs if `name` changes, instead of running it on every render. shouldn't be much an issue for now though, of course

Reply:
wrapped it with a `useMemo`

Reply:
that works, thanks!